lbplanner/services/user/delete_user.php

```diff
@@ -82,12 +83,12 @@ public static function delete_user($userid) {
         // Deleting all Courses associated with the User.
         $DB->delete_records(course_helper::LBPLANNER_COURSE_TABLE, array('userid' => $userid));
         // Deleting User from User table.
-        $DB->delete_records(user_helper::TABLE, array('userid' => $userid));
+        $DB->delete_records(user_helper::LB_PLANNER_USER_TABLE, array('userid' => $userid));
 
         return array('message' => 'User deleted successfully');
     }
 
-    public static function delete_user_returns() {
+    public static function delete_user_returns(): external_single_structure {
         return new external_single_structure(
             array('message' => new external_value(PARAM_TEXT, 'The message to return to the user'))
```

Comment:
this shouldn't return anything

Reply:
not even a message?

Reply:
nah, no exception is message enough :)

Reply:
i would like to have still a message as a response. Maybe as an integer?

Reply:
Even if the function doesn’t return meaningful data, it’s often a good idea to at least return a success status or a confirmation message. This can help with debugging and ensures that the client calling the function can verify that the function completed successfully.

Reply:
why return something that has no meaningful data. the client can verify if the function completed successfully via the HTTP status code and through the function not throwing an error. no error = success.

Reply:
> i would like to have still a message as a response. Maybe as an integer?

and what would the integer represent?